recipes/icu/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+cd source
+chmod +x configure install-sh
+
+if [ "$(uname)" == "Linux" ]; then
+    ./configure --prefix="$PREFIX"
```

Comment:
Once we have made these changes. There likely won't be a need for separate Linux and Mac sections (with the exceptions of what other environment variables need to be set). So, we probably can use the same configure step for both.